tools/triage/ticket_search/cli/command.py

```diff
@@ -0,0 +1,12 @@
+from workflow_data import WorkflowData
+
+
+class Command:
+    def run(self, workflow_data: "WorkflowData"):
+        pass
+
+    def get_command_key(self) -> str:
```

Comment:
since we don't need it dynamic, let's have is as a class member:
```
class Command:
    COMMAND_KEY = NotImplemented
    FRIENDLY_NAME = NotImplemented
```